components/dropzone/index.css

```diff
@@ -11,21 +11,191 @@ governing permissions and limitations under the License.
+  --spectrum-drop-zone-heading-to-body: var(--spectrum-spacing-75);
+
+  --spectrum-drop-zone-border-width: var(--spectrum-border-width-200);
+  --spectrum-drop-zone-corner-radius: var(--spectrum-corner-radius-100);
+  --spectrum-drop-zone-border-color: var(--spectrum-gray-300);
+
+  --spectrum-drop-zone-heading-font-family: var(--spectrum-sans-font-family-stack);
+  --spectrum-drop-zone-heading-font-weight: var(--spectrum-heading-sans-serif-font-weight);
+  --spectrum-drop-zone-heading-font-style: var(--spectrum-heading-sans-serif-font-style);
+  --spectrum-drop-zone-heading-font-size: var(--spectrum-drop-zone-title-size);
+  --spectrum-drop-zone-heading-line-height: var(--spectrum-heading-line-height);
+  --spectrum-drop-zone-heading-color: var(--spectrum-heading-color);
+
+  --spectrum-drop-zone-body-font-family: var(--spectrum-sans-font-family-stack);
+  --spectrum-drop-zone-body-font-weight: var(--spectrum-body-sans-serif-font-weight);
+  --spectrum-drop-zone-body-font-style: var(--spectrum-body-sans-serif-font-style);
+  --spectrum-drop-zone-body-font-size: var(--spectrum-drop-zone-body-size);
+  --spectrum-drop-zone-body-line-height: var(--spectrum-body-line-height);
+  --spectrum-drop-zone-body-color: var(--spectrum-body-color);
+
+
+  /* These need to be updated when we have support for recursive rgb values
+  --spectrum-drop-zone-background-color-rgb is a custom token */
+  --spectrum-drop-zone-background-color: var(--spectrum-drop-zone-background-color-rgb);
+  --spectrum-drop-zone-border-color-hover: var(--spectrum-accent-visual-color);
+  --spectrum-drop-zone-illustration-color: var(--spectrum-neutral-visual-color);
+  --spectrum-drop-zone-illustration-color-hover: var(--spectrum-accent-visual-color);
+
+
+  /* Filled styles */
+  --spectrum-drop-zone-content-height: var(--spectrum-component-height-300);
+  --spectrum-drop-zone-content-max-width: var(--spectrum-drop-zone-content-maximum-width);
+  --spectrum-drop-zone-content-edge-to-text: var(--spectrum-component-edge-to-text-300);
+  --spectrum-drop-zone-content-top-to-text: var(--spectrum-component-top-to-text-300);
+  --spectrum-drop-zone-content-bottom-to-text: var(--spectrum-component-bottom-to-text-300);
+
+  --spectrum-drop-zone-content-font-family: var(--spectrum-sans-font-family-stack);
+  --spectrum-drop-zone-content-font-weight: var(--spectrum-bold-font-weight);
+  --spectrum-drop-zone-content-font-style: var(--spectrum-default-font-style);
+  --spectrum-drop-zone-content-font-size: var(--spectrum-font-size-300);
+  --spectrum-drop-zone-content-line-height: var(--spectrum-line-height-100);
+
+  --spectrum-drop-zone-content-background-color: var(--spectrum-accent-visual-color);
+  --spectrum-drop-zone-content-color: var(--spectrum-white);
+
+
+  /* cjk language support */
+  &:lang(ja),
+  &:lang(zh),
+  &:lang(ko) {
+    --spectrum-drop-zone-heading-font-size: var(--spectrum-drop-zone-cjk-title-size);
+  }
+
+}
+
+.spectrum-Dropzone {
```

Comment:
just want to check on the casing here, should we use `.spectrum-DropZone`? 

I see that in the .`yml` file the name is `Drop Zone`, so two words, and other two word components we capitalize the second word in the class, like in DropIndicator, LogicButton, PickerButton etc.

Reply:
Good question. This was how it was when I picked up the card so I left it to avoid any conflicts with SWC but that should be a simple fix on their end in order to keep things consistent